solr_configs/catalog-production-v2/conf/schema.xml

```diff
@@ -190,11 +190,29 @@
      </fieldType>
 
     <fieldType name="text_lr" class="solr.TextField" positionIncrementGap="100">
-      <analyzer>
+      <analyzer type="index">
```

Comment:
Splitting this analyzer into an index analyzer and a query analyzer, so that we can use FlattenGraphFilterFactory at index time only per [solr's recommendation](https://lucene.apache.org/core/8_0_0/analyzers-common/org/apache/lucene/analysis/synonym/SynonymFilterFactory.html)